CTP/shell/src/com/navercorp/cubridqa/shell/main/TestFactory.java

```diff
@@ -82,24 +90,31 @@ private void addSkippedTestCases(ArrayList<String> list, String skippedType) {
 	}
 
 	public void execute() throws Exception {
-		
+		ArrayList<String> stableEnvList = (ArrayList<String>) context.getEnvList().clone();
 		if (context.isContinueMode) {
 			feedback.onTaskContinueEvent();
+			Dispatch.init(context);
+			if(Dispatch.getInstance().getTotalTbdSize() == 0){
+				System.out.println("Not found any test cases to execute with CONTINUE mode!");
+				return;
+			}
+			
+			System.out.println("============= SVN UPDATE ==================");
+			concurrentSVNUpdate(stableEnvList);
+			System.out.println("DONE");
+			
 		} else {
 			feedback.onTaskStartEvent(context.getCubridPackageUrl());
-		}
-		
-		ArrayList<String> stableEnvList = (ArrayList<String>) context.getEnvList().clone();
-		
-		System.out.println("============= SVN UPDATE ==================");
-		concurrentSVNUpdate(stableEnvList);
-		System.out.println("DONE");
-		
-		System.out.println("============= FETCH TEST CASES ==================");
-		Dispatch.init(context);
-		if(Dispatch.getInstance().getTotalTbdSize() == 0 ) {
-			System.out.println("NO TEST CASE TO TEST");			
-			return;
+			System.out.println("============= SVN UPDATE ==================");
+			concurrentSVNUpdate(stableEnvList);
+			System.out.println("DONE");
+			
+			System.out.println("============= FETCH TEST CASES ==================");
+			Dispatch.init(context);
+			if (Dispatch.getInstance().getTotalTbdSize() == 0) {
+				System.out.println("NO TEST CASE TO TEST!");
```

Comment:
System.out.println("Not found any test cases to execute with CONTINUE mode!");
System.out.println("NO TEST CASE TO TEST!");

It's better to have consistent output information.